app/components/scbd-angularjs-controls/form-control-directives/hover-effect.js

```diff
@@ -0,0 +1,26 @@
+define(['app','lodash','angular'], function(app,_,angular) {
+    app.directive('hoverEffect',['$timeout', function ($timeout  ) {
+    return {
+        scope: {
+            hoverEffect: '@'
+        },
+        restrict: 'A',
+        link: function (scope, element) {
+            if(scope.hoverEffect==undefined || scope.hoverEffect.length==0) return;
```

Comment:
`==` with `undefined` is not a good comparaison.  you need `===` to make good comparaison with undefined

Checking if `scope.hoverEffect` is falsy can do the job and easier to read
```suggestion
            if(!scope.hoverEffect) return;
```